contrib/init/systemd/docker.service

```diff
@@ -5,7 +5,8 @@ After=network.target docker.socket
 Requires=docker.socket
 
 [Service]
-ExecStart=/usr/bin/docker -d -H fd://
+EnvironmentFile=/etc/default/docker
```

Comment:
we have changed the docs to account for this https://github.com/docker/docker/pull/15301